examples/lock-app/esp32/main/AppTask.cpp

```diff
@@ -114,7 +114,7 @@ CHIP_ERROR AppTask::Init()
 void AppTask::AppTaskMain(void * pvParameter)
 {
     AppEvent event;
-    Clock::MonotonicMicroseconds mLastChangeTimeUS = 0;
+    Clock::Timestamp lastChangeTime{ 0 };
```

Comment:
Can we use System::Clock::Zero (I see it is used below)?

Reply:
Sure